pkg/jx/cmd/delete_team.go

```diff
@@ -161,7 +161,8 @@ func (o *DeleteTeamOptions) deleteTeam(name string) error {
 	}
 	o.changeNamespace(name)
 
-	err = o.ModifyTeam(name, func(team *v1.Team) error {
+	//TODO: will be wrong admin ns here.
```

Comment:
ahh... #2451 ?

Reply:
i'm just turning my attention to the delete one now - but yes, I suspect that is the cause of wrecking the admin namespace!